### PR TITLE
Create test functions for `editTodoItem`, `setCompleted`, `removeCompleted` 

### PR DESCRIPTION
**Pull request by MrBrN197**
### Changes
- Created test function for `editTodoItem`
- Created test function for `setCompleted`
- Created test function for `removeCompleted`

### CODE REVIEW

File: __test__/todo.test.js [hunk 1]
```diff
@@ -33,4 +40,99 @@ describe('DOM Manipulation', () => {
...
+    expect(storage.todoList.data).toHaveLength(2);
+
+    // Item 2 should remain and be unmodified
+    expect(storage.todoList.data[0].description).toBe('Item 2');
+    expect(storage.todoList.data[0].completed).toBe(false);
+
+    // Item 2 should remain and be unmodified
+    expect(storage.todoList.data[1].description).toBe('Item 3');
+    expect(storage.todoList.data[1].completed).toBe(false);
+  });
```
Comment (DeliceLydia) [inline __test__/todo.test.js]: - [x] Please add tests for the "Clear all completed" function.

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: __test__/todo.test.js (hunk 1 above)
Comment (MrBrN197) [reply __test__/todo.test.js]: Hi, I have the test I just forgot to add the name of the test. I will do this right away. Thank you.

**Review by Lameck1 — APPROVED** [review]
Hi, @MrBrN197 & @Rayan84,

Your project is complete!  It's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**